operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/AbstractConfigurationService.java

```diff
@@ -60,12 +54,14 @@ protected void throwExceptionOnNameCollision(
     final var key = keyFor(controller);
     final var configuration = configurations.get(key);
     if (configuration == null) {
-      log.warn(
-          "Configuration for controller '{}' was not found. {}", key, getControllersNameMessage());
+      logMissingControllerWarning(key, getControllersNameMessage());
```

Comment:
Just to my understanding, why is this better?

Reply:
The benefit is minimal, indeed, just to be able to not pull SLF4J when using `AbstractConfigurationService`.

Reply:
Hmm, isn't this a little bit over-engineering? We use SLF4J everywhere, and it's quite an industry standard

Reply:
The other issue is also that the previous version hardcoded the name of the logger, which might not have been appropriate depending on the actual implementation being executed (for example, even the Quarkus implementation would have logged something with `Default ConfigurationService implementation` logger name).